CHANGELOG.md

```diff
@@ -14,9 +14,13 @@ We use the following categories for changes:
 
 ## [Unreleased]
 
+## [0.5.3] - 2021-07-14
```

Comment:
```suggestion
## [0.5.3] - 2022-07-14
```

Reply:
Hehe. Shall we do the same for 0.5.2 and friends? :-)